domains/event-store/poc/Topics/Unpack.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using Newtonsoft.Json;
+using EventStore;
+using EventStore.Serialization;
+
+// FIXME: This namespace is hard to place
+
+namespace Topics;
+
+public class Unpack {
```

Comment:
For a more elegant solution we could look into using https://www.newtonsoft.com/json/help/html/SerializeTypeNameHandling.htm